doc/sphinx/source/ops/quantize.rst

```diff
@@ -0,0 +1,66 @@
+.. quantize.rst: 
+
+########
+Quantize
+########
+
+.. code-block:: cpp
+
+   Quantize // Maps real input to quantized output using scale, offset and round mode
+
+Description
+===========
+
+Produces a tensor of element type ``type`` and the same shape as ``input``
+where the value of each coordinate (i) of ``output`` is the corresponding coordinate of 
+``input`` divided by ``scale`` rounded as specified by ``round_mode`` minus ``offset``.  
+The coordinate (j) of ``scale`` and ``offset`` is the coordinate of ``output``
+projected along ``axes``.
+
+Inputs
+------
+
++-----------------+-------------------------+---------------------------------------+
+| Name            | Element Type            | Shape                                 |
++=================+=========================+=======================================+
+| ``input``       | is_real()               | Any                                   |
++-----------------+-------------------------+---------------------------------------+
+| ``scale``       | Same as ``input``       | ``input`` shape projected on ``axes`` |
++-----------------+-------------------------+---------------------------------------+
+| ``offset``      | Same as ``output``      | ``input`` shape projected on ``axes`` |
++-----------------+-------------------------+---------------------------------------+
+
+Attributes
+----------
+
++-------------------------------+----------------------------------------------------------------+
+| Name                          | Description                                                    |
++===============================+================================================================+
+| ``type``                      | ``output`` element type                                        |
```

Comment:
Would rephrase as "The output element, type, which must be a quantized type", and see below...